src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,48 @@
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    font-weight: 500;
+    font-size: 12px;
+    color: $color-main-accent;
+    text-transform: uppercase;
+  }
+
+  &__item {
+    margin-left: 20px;
+
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+
+  &__link {
+    position: relative;
+    display: flex;
+    align-items: center;
+    height: 60px;
```

Comment:
it's better not hardcode height for elements with content inside. Just use paddings